lib/tokyo_api/krautbuster.rb

```diff
@@ -25,12 +25,16 @@ def subscription_status(token)
       end
     end
 
-    def user_path(session_id, petition_id)
-      "/#{normalized_base_path}user/#{session_id}?petition_id=#{petition_id}"
+    def user_path(session_id, petition_id:, required_fields: nil)
+      path = "/#{normalized_base_path}user/#{session_id}?petition_id=#{petition_id}"
+      unless required_fields.nil?
+        path << "&required_fields=#{required_fields.join(',')}"
+      end
+      path
```

Comment:
We should create a Trello card for Campact to know how to deploy these changes, it should include the commit SHA of agra & tokyo to get the changes consistently deployed.

Reply:
Good point. Is there a particular column this card should go in?
I think they'll mainly need to deploy the updated tokyo before or at the same time as they deploy the updated agra.

Reply:
https://trello.com/c/v22wOhEA